app/src/main/res/layout/activity_options.xml

```diff
@@ -83,6 +84,15 @@
 			android:layout_weight="2" />
 	</LinearLayout>
 
+	<androidx.appcompat.widget.SwitchCompat
+		android:id="@+id/optionsOrderZeroItemsLast"
```

Comment:
On the other controls, the id refers to the type of control (e.g. `optionsThemeSwitch` or `optionsOrderDropdown`). For consistency this should probably be something like `optionsOrderZeroItemsLastSwitch` (long name, I know. Maybe you can think of something better)

Reply:
long name yes but internal and only used a few time. I personnaly do not encode the type in the name, it's checked by the compiler and should be obvious by the context but changing all the other IDs here (to be consistent) is a bit out of place IMO.
renamed to end id with "Switch" and string with "Text".